src/panels/lovelace/strategies/overview/overview-area-view-strategy.ts

```diff
@@ -71,113 +78,214 @@ export class OverviewAreaViewStrategy extends ReactiveElement {
+          ...climate.map(computeTileCard),
         ],
       });
     }
 
-    if (climate.length > 0) {
+    if (security.length > 0) {
       sections.push({
         type: "grid",
         cards: [
           computeHeadingCard(
-            hass.localize("ui.panel.lovelace.strategy.areas.groups.climate"),
-            AREA_STRATEGY_GROUP_ICONS.climate
+            "Security",
+            OVERVIEW_SUMMARIES_ICONS.security,
+            "security"
           ),
-          ...climate.map(computeTileCard),
+          ...security.map(computeTileCard),
         ],
       });
     }
 
-    if (media_players.length > 0) {
+    if (mediaPlayers.length > 0) {
       sections.push({
         type: "grid",
         cards: [
           computeHeadingCard(
-            hass.localize(
-              "ui.panel.lovelace.strategy.areas.groups.media_players"
-            ),
-            AREA_STRATEGY_GROUP_ICONS.media_players
+            "Media players",
+            OVERVIEW_SUMMARIES_ICONS.media_players,
+            "media-players"
           ),
-          ...media_players.map(computeTileCard),
+          ...mediaPlayers.map(computeTileCard),
         ],
       });
     }
 
-    if (security.length > 0) {
-      sections.push({
-        type: "grid",
-        cards: [
-          computeHeadingCard(
-            hass.localize("ui.panel.lovelace.strategy.areas.groups.security"),
-            AREA_STRATEGY_GROUP_ICONS.security
-          ),
-          ...security.map(computeTileCard),
-        ],
+    const deviceSections: LovelaceSectionRawConfig[] = [];
+
+    const summaryEntities = Object.values(entitiesBySummary).flat();
+
+    // Rest of entities grouped by device
+    const otherEntities = areaEntities.filter(
+      (entityId) => !summaryEntities.includes(entityId)
+    );
+
+    const entitiesByDevice: Record<string, string[]> = {};
+    const unassignedEntities: string[] = [];
+    for (const entityId of otherEntities) {
+      const stateObj = hass.states[entityId];
+      if (!stateObj) continue;
+      const { device } = getEntityContext(stateObj, hass);
+      if (!device) {
+        unassignedEntities.push(entityId);
+        continue;
+      }
+      if (!(device.id in entitiesByDevice)) {
+        entitiesByDevice[device.id] = [];
+      }
+      entitiesByDevice[device.id].push(entityId);
+    }
+
+    const otherDeviceEntities = Object.entries(entitiesByDevice).map(
+      ([deviceId, entities]) => ({
+        device_id: deviceId,
+        entities: entities,
+      })
+    );
+
+    if (unassignedEntities.length > 0) {
+      otherDeviceEntities.push({
+        device_id: "unassigned",
+        entities: unassignedEntities,
       });
     }
 
-    if (actions.length > 0) {
-      sections.push({
+    const batteryFilter = generateEntityFilter(hass, {
+      domain: "sensor",
+      device_class: "battery",
+    });
+
+    const energyFilter = generateEntityFilter(hass, {
+      domain: "sensor",
+      device_class: ["energy", "power"],
```

Comment:
@piitaya I understand that `energy` is filtered out, because it's shown by the dedicated Energy dashboard (if configured accordingly), but why are the `power` entities filtered out too? I have a couple of smart plugs and what I can see from them in the dashboard are the voltage and current, but the more interesting power values are missing.